apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/Internal/Logging/Logger.cs

```diff
@@ -193,7 +193,7 @@ public virtual void Error(Func<string> messageFunc, Exception exception = null)
         {
             if (LogLevel >= LogLevel.Error)
             {
-                WriteLine(messageFunc());
+                WriteLine($"{messageFunc()}, Exception = {exception}");
```

Comment:
Presumably the rest of the changes in Google.Cloud.Spanner.V1 will go away after #1650 is merged?

Reply:
yep